src/components/AppHeader.jsx

```diff
@@ -12,24 +19,40 @@ import { useSession } from '@inrupt/solid-ui-react';
 const AppHeader = () => {
   const { session } = useSession();
 
-  const headerStyle = {
-    width: '100%',
-    gap: '20px',
-    display: 'flex',
-    alignItems: 'center',
-    justifyContents: 'center'
-  };
-
   return (
-    <header style={headerStyle}>
-      <h2>Getting Started with PASS</h2>
-      {session.info.isLoggedIn ? (
-        <>
-          <Link to="/PASS/home/">Home</Link>
-          <Link to="/PASS/forms/">Forms</Link>
-        </>
-      ) : null}
-    </header>
+    <>
+      <CssBaseline />
+      <Box sx={{ flexGrow: 1 }}>
+        <AppBar position="static" color="primary">
+          <Toolbar>
+            {session.info.isLoggedIn ? (
+              <>
+                <Typography variant="h4" noWrap component="div">
+                  PASS
+                </Typography>
+                <ButtonGroup
```

Comment:
I do think we want to cluster the links, but I don't think we need a ButtonGroup.

Reply:
> I also thought we're transitioning to Navbar and replacing the AppHeader after our recent discussions?

The code has already been consolidated. Do you mean changing the file structure and moving it into its own folder?